util/package.json

```diff
@@ -4,10 +4,12 @@
   "description": "Script to help build required JSON from Excel",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "convert": "node convert_to_json.js && node json_to_data_dir.js"
   },
   "author": "Vatsal Joshi",
   "dependencies": {
-    "xls-to-json": "^0.4.0"
+    "xls-to-json": "^0.4.0",
+    "xlsx": "^0.13.1"
```

Comment:
Was this extra package needed? I think `xls-to-json` just does the work.

Reply:
It was returning an array with an empty object. I converted the PDF via https://smallpdf.com/pdf-to-excel and that module couldn't parse it properly. These scripts will change because of VIT's inconsistent means of storing the report. Maybe it's better to split it up into modules and do all the work in `index.js`. I just wanted to test and bring it out quickly.